contrib/clojure-package/examples/bert/src/bert/bert_sentence_classification.clj

```diff
@@ -157,4 +183,42 @@
 (comment
 
   (train (context/cpu 0) 3)
-  (m/save-checkpoint model {:prefix "fine-tune-sentence-bert" :epoch 3}))
+
+  (m/save-checkpoint model {:prefix fine-tuned-prefix :epoch 3})
+
+  
+  ;;;; Explore results from the fine-tuned model
+
+  ;; We need a predictor with a batch size of 1, so we can feed the
+  ;; model a single sentence pair.
+  (def fine-tuned-predictor
+    (infer/create-predictor (infer/model-factory fine-tuned-prefix
+                                                 [{:name "data0" :shape [1 seq-length] :dtype dtype/FLOAT32 :layout layout/NT}
+                                                  {:name "data1" :shape [1 seq-length] :dtype dtype/FLOAT32 :layout layout/NT}
+                                                  {:name "data2" :shape [1]            :dtype dtype/FLOAT32 :layout layout/N}])
+                            {:epoch 3}))
+  
+  ;; Get the fine-tuned model's opinion on whether two sentences are equivalent:
+  (defn predict-equivalence
+    [predictor sentence1 sentence2]
+    (let [vocab (bert.util/get-vocab)
+          processed-test-data (mapv #(pre-processing (:idx->token vocab)
+                                                     (:token->idx vocab) %)
+                                    [[sentence1 sentence2]])
+          prediction (infer/predict-with-ndarray predictor
+                                                 [(ndarray/array (slice-inputs-data processed-test-data 0) [1 seq-length])
+                                                  (ndarray/array (slice-inputs-data processed-test-data 1) [1 seq-length])
+                                                  (ndarray/array (slice-inputs-data processed-test-data 2) [1])])]
+      (ndarray/->vec (first prediction))))
+
+  ;; Modify an existing sentence pair to test:
+  ;; ["1"
+  ;;  "69773"
+  ;;  "69792"
+  ;;  "Cisco pared spending to compensate for sluggish sales ."
+  ;;  "In response to sluggish sales , Cisco pared spending ."]
+  (predict-equivalence fine-tuned-predictor
```

Comment:
did you want to add a test for this?

Reply:
Can do

Reply:
Just pushed one. Thanks for the idea—my PR broke the existing test, but I guess tests for examples aren't part of the CI checks.